components/brave_new_tab_ui/components/search/config.ts

```diff
@@ -2,10 +2,13 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { loadTimeData } from '$web-common/loadTimeData'
 import { radius } from "@brave/leo/tokens/css/variables"
 
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 export const ENABLED_SEARCH_ENGINES_KEY = 'search-engines'
 export const LAST_SEARCH_ENGINE_KEY = 'last-search-engine'
-export const braveSearchHost = 'search.brave.com'
```

Comment:
Think we might need to export both for the SearchBox placeholder

Reply:
Done.